qemu/tests/ovs_host_vlan.py

```diff
@@ -0,0 +1,249 @@
+                output = utils.system_output(create_port_cmd, timeout=120,
+                                             ignore_status=False)
+            except error.CmdError:
+                err = "Fail to create ovs port %s on bridge %s." % (ifname,
+                                                                    bridge_name)
+                err += " Command: %s, output: %s." % (create_port_cmd, output)
+                raise error.TestFail(err)
+
+            session_ctl = vm.wait_for_serial_login(timeout=login_timeout)
+            txt = "Stop NetworkManager service in guest %s." % vm_name
+            session_ctl.cmd("service NetworkManager stop",
+                            ignore_all_errors=True)
+
+            txt = "Set guest %s IP to %s" % (vm_name, guest_ip)
+            error.context(txt, logging.info)
+            mac = vm.get_mac_address()
+            utils_net.set_guest_ip_addr(session_ctl, mac, guest_ip,
+                                        os_type=os_type)
+            ips.append(guest_ip)
+            sessions.append(session_ctl)
+
+        txt = "Ping between two guests in same vlan. %s -> %s" % (vms[0],
+                                                                  vms[1])
+        error.context(txt, logging.info)
+        ping(ips[1], count=100, session=sessions[0], same_vlan=True,
+             os_type=os_type)
+
+        txt = "Ping between two guests in different vlan. %s -> %s" % (vms[0],
+                                                                       vms[2])
+        error.context(txt, logging.info)
+        ping(ips[2], count=100, session=sessions[0], same_vlan=False)
+
+        txt = "Ping between two guests in another vlan. %s -> %s" % (vms[2],
+                                                                     vms[3])
+        error.context(txt, logging.info)
+        ping(ips[2], count=100, session=sessions[3], same_vlan=True)
+
+        txt = "Netperf test between two guests in same vlan."
+        txt += "%s -> %s" % (vms[0], vms[1])
+        error.context(txt, logging.info)
+
+        txt = "Run netserver in VM %s" % vms[0]
+        error.context(txt, logging.info)
+        netserver_cmd = params.get("netserver_cmd")
+        netperf_path = params.get("netperf_path")
+        cmd = os.path.join(netperf_path, netserver_cmd)
+        status, output = sessions[0].cmd_status_output(cmd, timeout=60)
+        if status != 0:
+            err = "Fail to start netserver in VM."
+            err += " Command output %s" % output
+            raise error.TestError(err)
+
+        txt = "Run netperf client in VM %s" % vms[1]
+        error.context(txt, logging.info)
+        test_duration = int(params.get("netperf_test_duration", 60))
+        test_protocol = params.get("test_protocol")
+        netperf_cmd = params.get("netperf_cmd")
+        netperf_cmd = os.path.join(netperf_path, netperf_cmd)
+        cmd = netperf_cmd % (test_duration, ips[0])
+        if test_protocol:
```

Comment:
@ we have  -t %s in  qemu/tests/cfg/ovs_host_vlan.cfg :line27, so here will get invaild netperf command